sagemaker-debugger/pytorch_profiling/pt-resnet-profiling-single-gpu-single-node.ipynb

```diff
@@ -9,14 +9,37 @@
     "This notebook will walk you through creating a PyTorch training job with the SageMaker Debugger profiling feature enabled. It will create a single GPU single node training.\n"
```

Comment:
<pre class="ql-syntax" spellcheck="false">import sys
import IPython
install_needed = True  # should only be True once
if install_needed:
    print("installing deps and restarting kernel")
    !{sys.executable} -m pip install -U sagemaker
    !{sys.executable} -m pip pip install -U smdebug
    IPython.Application.instance().kernel.do_shutdown(True)
</pre><p> </p><p>would you update this throughout the all other notebooks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>